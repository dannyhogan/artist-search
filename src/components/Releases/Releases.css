@import url('https://fonts.googleapis.com/css?family=Montserrat&display=swap');

.Releases {

  width: 90vw;
  display: flex;
  flex-wrap: wrap;
  list-style: none;
  padding: 0;
  font-family: 'Montserrat';

  li {
    a {
      padding: 10px;
      border: 2px solid rgb(213, 213, 213);
      border-radius: 15px;
      background: rgba(0, 0, 0, 0.12);
      margin: 10px;
      min-width: 200px;
      height: 200px;
      display: flex;
      flex-direction: column;
      justify-content: space-between;
      align-items: center;
      text-decoration: none;
      color: rgb(39, 39, 39);
      letter-spacing: 0px;
    }

    img {
      width: 100px;
      max-width: 100px;
    }

    h2 {
    }
  }
}
