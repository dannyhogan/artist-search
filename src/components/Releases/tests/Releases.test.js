import React from 'react';
import { shallow } from 'enzyme';
import Releases from '../Releases';

describe('Releases component', () => {
  it('renders Releases component', () => {
    const release = { title: 'Forever', id: '123', 'cover-art-archive': { count: 3 } };
    const wrapper = shallow(<Releases artist="Drake" releases={[release]} />);
    expect(wrapper).toMatchSnapshot();
  });
});
