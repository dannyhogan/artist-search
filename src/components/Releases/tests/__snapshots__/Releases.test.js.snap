// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Releases component renders Releases component 1`] = `
<ul
  className="Releases"
>
  <li
    key="123"
  >
    <Link
      to="/release/Drake/Forever/123"
    >
      <img
        src="http://coverartarchive.org/release/123/front"
      />
      <h2>
        Forever
      </h2>
      <p />
    </Link>
  </li>
</ul>
`;
